test/Microsoft.ML.Predictor.Tests/TestPredictors.cs

```diff
@@ -1058,21 +1058,26 @@ public void CalibratorPerceptronTest()
         }
 
         /// <summary>
-        ///A test for calibrators
+        ///A test for default calibrators
         ///</summary>
-        [Fact(Skip = "Need CoreTLC specific baseline update")]
+        [Fact]
         [TestCategory("Calibrator")]
-        public void CalibratorLinearSvmTest()
+        public void NoCalibratorLinearSvmTest()
         {
             var datasets = GetDatasetsForCalibratorTest();
-            RunAllTests(
```

Comment:
This looks like a positive change. I wonder if we have an opinion of what we're going to do for those tests though that have a more elaborate set of datasets and configuration options. In those cases the cartesian product would be a bit more elaborate.